src/components/Equity/EquityLpRow.tsx

```diff
@@ -0,0 +1,135 @@
+} from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import qs from 'qs'
+import React, { useCallback, useMemo } from 'react'
+import { useHistory, useLocation } from 'react-router'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { trackOpportunityEvent } from 'lib/mixpanel/helpers'
+import { MixPanelEvents } from 'lib/mixpanel/types'
+import type { OpportunityId } from 'state/slices/opportunitiesSlice/types'
+import { getUnderlyingAssetIdsBalances } from 'state/slices/opportunitiesSlice/utils'
+import {
+  selectAllEarnUserLpOpportunitiesByFilter,
+  selectAssetById,
+  selectAssets,
+  selectMarketDataSortedByMarketCap,
+  selectOpportunityApiPending,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { EquityRow } from './EquityRow'
+
+type EquityLpRowProps = {
+  opportunityId: OpportunityId
+  assetId: AssetId
+  accountId?: AccountId
+  totalFiatBalance?: string
+  color?: string
+}
+export const EquityLpRow: React.FC<EquityLpRowProps> = ({
+  opportunityId,
+  assetId,
+  accountId,
+  totalFiatBalance,
+  color,
+}) => {
+  const {
+    state: { isConnected, isDemoWallet },
+    dispatch,
+  } = useWallet()
+  const history = useHistory()
+  const location = useLocation()
+  const isLoading = useAppSelector(selectOpportunityApiPending)
+  const assets = useAppSelector(selectAssets)
+  const marketData = useAppSelector(selectMarketDataSortedByMarketCap)
+  const filter = useMemo(() => {
+    return {
+      assetId,
+      ...(accountId ? { accountId } : {}),
+    }
+  }, [accountId, assetId])
+  const lpOpportunities = useAppSelector(state =>
+    selectAllEarnUserLpOpportunitiesByFilter(state, filter),
+  )
+  const opportunity = lpOpportunities.find(opportunity => opportunity.id === opportunityId)
+  const asset = useAppSelector(state => selectAssetById(state, assetId))
+
+  if (!opportunity) throw new Error(`No opportunity found for ${assetId}`)
+  if (!assetId) throw new Error(`No assetId ${assetId}`)
+
+  const underlyingBalances = getUnderlyingAssetIdsBalances({
+    underlyingAssetIds: opportunity.underlyingAssetIds,
+    underlyingAssetRatiosBaseUnit: opportunity.underlyingAssetRatiosBaseUnit,
+    cryptoAmountBaseUnit: opportunity.cryptoAmountBaseUnit,
+    assetId,
+    assets,
+    marketData,
+  })
+
+  const handleClick = useCallback(() => {
+    if (!opportunity) return
+    const {
+      type,
+      provider,
+      contractAddress,
+      chainId,
+      rewardAddress,
+      assetId,
+      highestBalanceAccountAddress,
+    } = opportunity
+    const { assetReference, assetNamespace } = fromAssetId(assetId)
+
+    if (!isConnected && isDemoWallet) {
+      dispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
+      return
+    }
+
+    trackOpportunityEvent(
+      MixPanelEvents.ClickOpportunity,
+      {
+        opportunity,
+        element: 'Table Row',
```

Comment:
```suggestion
        element: 'Equity LP Row',
```

just for more granularity?